style.css

```diff
@@ -185,13 +550,12 @@ body {
 	width: 35px;
 	border-radius: 50%;
 	margin-right: 20px;
-	--circle-color:blue;
+	--circle-color: blue;
```

Comment:
> Css variable: you defined a variable for the circle background color. What is the purpose? Being more explicit?

It was just to show you that you can define default values to variables if you need to. In this case, you have a modifier class for each variable, to go along with the block class, so it's okay to not use it :)